app/forms/apo_form.rb

```diff
@@ -83,6 +83,8 @@ def collection_radio
                   uri: 'http://opendatacommons.org/licenses/by/1.0/' },
     'odc-odbl' => { label: 'Open Data Commons Open Database License 1.0',
                     uri: 'http://opendatacommons.org/licenses/odbl/1.0/' },
+    'cc0' => { label: 'No Rights Reserved',
+               uri: 'https://creativecommons.org/share-your-work/public-domain/cc0/' },
```

Comment:
they seem to refer to the same license, ultimately, but the ticket does use a different URI:  https://creativecommons.org/publicdomain/zero/1.0/